stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-7.14
+resolver: lts-8.5
```

Comment:
Oh, I wasn't able to build with `lts-7.14` because `base >= 4.9` is required. Should I remove this from the PR?